SeaPkg/Tools/GenSeaArtifacts/gen_rim/src/lib.rs

```diff
@@ -0,0 +1,464 @@
+//! A library containing necessary data structures representing CBOR formatted

```

Comment:
Can you please add a copyright?

Reply:
Added to this, and multiple other files.